test/test_external_libs.py

```diff
@@ -1,10 +1,7 @@
 import unittest
-import os
 from test.util import ykman_cli
 
 
-@unittest.skipIf(
-    os.environ.get('INTEGRATION_TESTS') != 'TRUE', 'INTEGRATION_TESTS != TRUE')
```

Comment:
Are you sure about this? Shouldn't the python only unittests be able to run without the libraries in place?

Reply:
Oooh, now I see why this was there. Good catch, thanks! This should just be updated to use the new environment variable, then.

Reply:
Case in point: all CI build failed... 😁